torch/csrc/utils/schema_info.cpp

```diff
@@ -264,6 +300,24 @@ void SchemaInfo::generateAliasMaps() {
     }
   }
 
+  // Fills wildcard_set with container created wildcards.
+  // ie if one argument is a container and it contains an element that aliases
+  // the other argument, then the second argument is added to the wildcard set.
```

Comment:
an example would be nice to help understand this! You can probably just copy one out of the tests. (I was a bit confused until I read the tests)